src/material/progress-bar/progress-bar.scss

```diff
@@ -35,4 +50,15 @@
       transition: transform 1ms;
     }
   }
+
+  --mdc-linear-progress-primary-half: 83.67142%;
+  --mdc-linear-progress-primary-full: 200.611057%;
+  --mdc-linear-progress-secondary-quarter: 37.651913%;
+  --mdc-linear-progress-secondary-half: 84.386165%;
+  --mdc-linear-progress-secondary-full: 160.277782%;
+  --mdc-linear-progress-primary-half-neg: -83.67142%;
+  --mdc-linear-progress-primary-full-neg: -200.611057%;
+  --mdc-linear-progress-secondary-quarter-neg: -37.651913%;
+  --mdc-linear-progress-secondary-half-neg: -84.386165%;
+  --mdc-linear-progress-secondary-full-neg: -160.277782%;
```

Comment:
I'm not sure what to do about these css variables that were previously getting set to some default values and otherwise can only be configured through an adapter. Should I file a bug with MDC or is it ok to just set these this way?

Reply:
You should add a comment about where they are derived from, and how we can eventually get rid of them. Yes file a bug and ideally we can put it in the comment e.g. b/1234789